Kosaraju_Algorithm/Kosaraju_algorithm.dart

```diff
@@ -0,0 +1,220 @@
+/**
+ *   Kosaraju's Algorithm
+ * ----------------------------------------
+ * 
+ * This algorithm is used to find the number of strongly connected components
+ * in a directed graph. It uses DFS. It performs DFS twice to achieve the goal based on the fact that
+ * finishing time of starting vertex will be greater than the finishing times of vertices in other SCCs.
+ * 
+ * References - Geeksforgeeks
+ * 
+ */
+
+
```

Comment:
remove this extra lines

Reply:
I removed them and committed. I also squashed the commit.